test/spec/app_spec.rb

```diff
@@ -783,10 +797,51 @@
     #   skip("Needs to be implemented")
     # end
 
-    # it "should edit team logo" do
-    #   skip("Needs to be implemented")
-    # end
+    it "should edit team and logo", bin1: true do
+      team = "testteam#{Time.now.to_i}"
+      api_create_team(team:team)
+      p = Page.new(config: @config, driver: @driver)
+      p.go(@config['self_url'] + '/' + team)
+      wait_for_selector("team-members__edit-button", :class)
+      str= @driver.page_source
+      str = str[str.index('team__edit-button')..str.length]
+      str = str[0..str.index('tabindex') - 3]
+      str = "." +str.gsub(" ", ".")
+      el = wait_for_selector(str)
+      el.click
 
+      el = wait_for_selector("team__name-container", :id)
+      el.click
+      el.send_keys "EDIT"
+
+      el = wait_for_selector("team__description-container", :id)
+      el.click
+      el.send_keys "EDIT DESCRIPTION"
+
+      el = wait_for_selector("team__location-container", :id)
+      el.click
+      el.send_keys "EDIT DESCRIPTION"
+
+      el = wait_for_selector("team__phone-container", :id)
+      el.click
+      el.send_keys "555199889988"
+
+      el = wait_for_selector("team__link-container", :id)
+      el.click
+      el.send_keys "www.meedan.com"
+
+      #Logo
+      str= @driver.page_source
+      str = str[str.index('team__edit-avatar-button')..str.length]
+      str = str[0..str.index('>') - 2]
+      str = "." +str.gsub(" ", ".")
+      el = wait_for_selector(str)
```

Comment:
Same as before: what's the idea of this block?

Reply:
Same answer as before.
Get the name of a button without text or unique id.